beater/cloudbeat.go

```diff
@@ -164,7 +167,9 @@ func (bt *cloudbeat) Run(b *beat.Beat) error {
 	}
 
 	output := bt.data.Output()
+	go bt.transformer.ProcessAggregatedResources(bt.ctx, output)
```

Comment:
An Improvement will be to run a loop that spawns as many Go routines as we have CPU cores available.